api/app/src/main/kotlin/packit/model/Metadata.kt

```diff
@@ -0,0 +1,28 @@
+package packit.model
+
+data class PacketMetadata(
```

Comment:
Can we make these model names consistent, so either just call this one `Packet`, or add `Metadata` to the others, i.e. `GitMetadata`, `FileMetadata` etc.?